packages/components/src/input-control/types.ts

```diff
@@ -191,6 +191,10 @@ export interface InputControlProps
 			| 'paddingInlineEnd'
 		> {
 	__unstableStateReducer?: InputFieldProps[ 'stateReducer' ];
+	/**
+	 * An additional description for the control.
+	 */
+	help?: string;
```

Comment:
`BaseControl`'s `help` prop is a `ReactNode` — is there any specific reason why we're using `string` instead of `Pick< BaseControlProps, 'help' >` ?

Reply:
This was because the `help` implementation in BaseControl was not marked up semantically, as @tellthemachines noted. ARIA descriptions actually require plain text. I was kind of considering removing `ReactNode` support in BaseControl itself in the process of fixing the markup issue.

However, taking a better look at the codebase, there are already a good number of `help` usage that include links in the text. So I'm going to suggest that we mark it up as `aria-describedby` when plain text, and fall back to `aria-details` when not. e4806bb

I believe it's still preferable to use `aria-describedby` because [`aria-details` support](https://www.digitala11y.com/aria-detailsproperties/) is pretty bad.

Reply:
Agreed, thank you for the context! Should we mention that a plain `string` of text is preferable in the `help` prop description in `BaseControl` ?

Reply:
Ok, I'll add that in the follow-up when I add the semantics to BaseControl 👍